### DISCUSSION TITLE
`Updating branches and 0 commit authors…`

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#81717] [Repositories]
### Select Topic Area

Product Feedback

### Body

When performing an import, one of the phases has:

```
 Updating branches and 0 commit authors…
```

The `0 commit authors` bit is a bit suboptimal.

I'm fairly certain that the repository I'm importing has more than one commit and thus more than 0 commit authors. And this message stays visible for a nontrivial amount of time (long enough for me to open this discussion).

**Comment by On-Fi** [comment]
I use the GitHub Desktop app to add my co-authors to every commit.

Did your co-author accept your invitation to work with you on the repo?

Hopefully this link can help you out: 
https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

**Reply by jsoref** [reply]
No. This is part of the repository import process, it isn't an in the wild thing.

**Reply by On-Fi** [reply]
Oh okay, than good luck with finding the real answer to you question
